shell/platform/windows/platform_handler_unittests.cc

```diff
@@ -61,12 +64,13 @@ TEST(PlatformHandler, GettingTextCallsThrough) {
   TestBinaryMessenger messenger;
   TestPlatformHandler platform_handler(&messenger);
 
-  auto args = std::make_unique<rapidjson::Document>(rapidjson::kArrayType);
-  auto& allocator = args->GetAllocator();
-  args->PushBack(kTextPlainFormat, allocator);
-  auto encoded = JsonMethodCodec::GetInstance().EncodeMethodCall(
-      MethodCall<rapidjson::Document>(kGetClipboardDataMethod,
-                                      std::move(args)));
+  std::ostringstream jsonStringStream;
+  jsonStringStream << "{\"method\":\"" << kGetClipboardDataMethod 
+      << "\",\"args\":\"" << kTextPlainFormat << "\"}";
+  std::string jsonString = jsonStringStream.str();
+  unsigned char json [256];
+  std::copy(jsonString.begin(), jsonString.end(), json);
+  unsigned char* data = &json[0];
```

Comment:
I couldn't for the life of me get the original deleted code to work.  The arguments seemed to be an array of an array, and rapidjson would fail to get them.  Notice that other tests have their arguments as an object, and that works.  Manually creating the json string like this also works, but if anyone knows how to get the original way to work, let me know.

Reply:
What was the error?